### ISSUE TITLE
Train/Dev sets speaker contamination in egs/ami/s5/local/split_*.orig 

### ISSUE THREAD

**Issue by nikosfl** [kaldi-asr/kaldi#3362]
Speakers FIE038 and MIO036 exist in both train and dev sets. This can be avoided by discarding completely the session IB4005 as suggested in http://groups.inf.ed.ac.uk/ami/corpus/datasets.shtml. Even though the splits in kaldi recipe in general follow the Full-corpus-ASR partition of this page, IB4005 exists in the training partition, causing the contamination.

**Comment by danpovey** [comment]
Hm.  I suppose this is a bug, but I really don't want to change it at this point, since it is after all the  'official' test/train split, and I doubt it would really make that much difference in practice.  We'll see what others think though.

**Comment by nikosfl** [comment]
I agree it is probably not worth it to change it, but people should just be aware of this; maybe just a mention in the README file of the recipe. 
(btw the page I mentioned is the official amicorpus page, so the kaldi split is the 'official' with the exception of this IB4005 session)

**Comment by danpovey** [comment]
If it's the official partition now, it might make sense to change it.
But I'd rather wait till someone has time to run the setup and update the
results (at least the more important ones).

**Comment by danpovey** [comment]
If someone could make a PR for this is, I could try to get it tested.
